src/rpcwallet.cpp

```diff
@@ -2481,16 +2481,31 @@ UniValue listspentzerocoins(const UniValue& params, bool fHelp)
 
 UniValue mintzerocoin(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() != 1)
+    if (fHelp || params.size() < 1 || params.size() > 2)
         throw runtime_error(
-            "mintzerocoin <amount>\n"
-            "Usage: Enter an amount of Piv to convert to zPiv"
+            "mintzerocoin <amount> [UTXOs]\n"
+            "amount: Enter an amount of Piv to convert to zPiv\n"
+            "UTXOs: (string, optional) A json array of objects. Each object the txid (string) vout (numeric)\n"
+            "     [           (json array of json objects)\n"
+            "       {\n"
+            "         \"txid\":\"id\",    (string) The transaction id\n"
+            "         \"vout\": n         (numeric) The output number\n"
+            "       }\n"
+            "       ,...\n"
+            "     ]\n"
             + HelpRequiringPassphrase());
```

Comment:
would like to see the inclusion of the `HelpExample*` functions here, something like 

```
HelpExampleCli("mintzerocoin", 50 "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\"") + HelpExampleRpc("mintzerocoin", 50 "\"[{\\\"txid\\\":\\\"myid\\\",\\\"vout\\\":0}]\"");
```